content/use-cases/contribute-code-to-ripple-lib.md

```diff
@@ -0,0 +1,69 @@
+# Contribute Code to ripple-lib
+
+Want to contribute code or a bug report to help improve `ripple-lib`, the official client library for [RippleAPI](rippleapi-reference.html)? RippleAPI is a JavaScript API for interacting with the XRP Ledger. Here’s a roadmap to the high-level tasks that’ll have you reviewing code and functionality in no time. ***TODO: What do you think of the use of ripple-lib vs RippleAPI throughout this use case? Are these being used correctly?***
+
+
+{% set n = cycler(* range(1,99)) %}
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## <a href="https://github.com/ripple/ripple-lib" target="_blank">Access the `ripple-lib` repo <i class="fa fa-external-link" aria-hidden="true"></i></a> <!--#{ fix for md highlighting_ #}-->
+
+`ripple-lib` is an open-source project. You can take a look at `ripple-lib` code simply by accessing the `ripple-lib` GitHub repo. Before contributing or reporting bugs, we recommend that you get to know the code and developer experience by performing the following tasks.
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## [Set up and run a `rippled` server](manage-the-rippled-server.html)
+
+RippleAPI is an API for interacting with the XRP Ledger. The core peer-to-peer server that manages the XRP Ledger is `rippled`. Set up and run a `rippled` server to understand its developer experience and functionality. Anyone can run their own `rippled` server that follows the network and keeps a complete copy of the XRP Ledger.
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## [Get a Test Net XRP Ledger account](xrp-test-net-faucet.html)
+
+Use the XRP Ledger Test Net to get a test account. Connect your `rippled` server to the Test Net to make test calls and get to know the XRP Ledger.
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+<!-- <span class="use-case-step-length">(1 hour)</span> -->
+## [Set up your `ripple-lib` development environment](get-started-with-rippleapi-for-javascript.html#environment-setup)
+
+`ripple-lib` requires Node.js and a few dependencies that you can get using npm (Node Package Manager). Also, be sure to create your own fork of the `ripple-lib` repository on GitHub so you can contribute pull requests to the official repo.
```

Comment:
I recommend using [Node.js v10 LTS](https://nodejs.org/) and [Yarn](https://yarnpkg.com/).